### PR TITLE
Adding the selenium basics with the python for beginners.

### PR DESCRIPTION
**Pull request by mohitpeshwani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -396,7 +396,9 @@ Distributed under the MIT LICENSE license. See [``LICENSE``](https://github.com/
...
+
+### If you like the project give a star  [<img src="Selection_008.png" alt="Star button" align="top">](https://github.com/Logan1x/Python-Scripts/stargazers)
```
Comment (Logan1x) [inline README.md:401]: Looks like this line is duplicated. Can you please remove it and let me know?

**Review by Logan1x — APPROVED** [review]

**Comment by Logan1x** [comment]
All other changes are looking good. Just change that line and lmk.

**Comment by Logan1x** [comment]
Thanks, @mohitpeshwani for contributing to this repository. I do not have any Active Flutter or Kotlin projects but I do have some react projects. If you would Like to see, please visit this link : https://logan1x.github.io/projects.html

**Comment by mohitpeshwani** [comment]
Cool, thanks!

**Pull request merged**